infer/src/base/DBWriter.ml

```diff
@@ -385,7 +385,9 @@ module Server = struct
         send Command.Handshake
 end
 
-let use_daemon = Config.((not (buck || genrule_mode)) && jobs > 1)
+let use_daemon =
+  Config.((not (buck || genrule_mode)) && jobs > 1 && not (String.equal Sys.os_type "Win32"))
```

Comment:
Would branching on Version.build_platform instead make sense?

nit: the windows test seems to be under the Config local open but doesn't need to be.

Reply:
That's an interesting question. Note that `build_platform` does not support Cygwin, and I suspect the daemon probably works there. OTOH more changes are probably needed to get the Cygwin version to work :sweat_smile: So, will change.

Reply:
Done.

Reply:
No longer relevant, as will disappear in the final rewrite.